src/lib/ui/ProNotebookMember/_deleteNotebookAppointment.gql

```diff
@@ -0,0 +1,5 @@
+mutation DeleteNotebookAppointmentById($id: uuid!) {
```

Comment:
If I understand the hasura API correctly, this performs a hard delete of the appointment.

Is that something we want to do? Performing a soft-delete (aka add a deletion date and author; and filter based on that info), would improve our ability to understand what happened and serve support requests.

Reply:
Effectivement, je me suis posé la question en l'implémentant, mais j'ai vu que les autres suppressions font un `hard delete`.
A discuter en équipe, je serai pour implémenter un `soft delete` en ajoutant un `is-deleted`, une `deletion-date` et éventuellement un `deletion-author-id`.

Reply:
Fait dans les 2 derniers commits 👍🏾